docker/checkout.bash

```diff
@@ -107,3 +107,7 @@ cd V2X-Hub
 git config core.sparsecheckout true
 git sparse-checkout init
 git sparse-checkout set ext/ccserver
+sudo apt-get update && sudo apt-get install -y libdlib-dev
```

Comment:
Since dlib is built as part of the `multiple_object_tracking/scripts/install_dependencies.sh`, I think we can remove this.